includes/api/ApiQueryRecentChanges.php

```diff
@@ -525,6 +531,27 @@ private function parseRCType( $type ) {
 		}
 	}
 
+	private function combineAdditionalData( ApiResult $result, array $revisionsIds ) {
```

Comment:
Do we expect to combine more data? It's minor, but just thinking whether this would be a little better as `addUpvoteData` or similar to be specific as to what it does at this time, and move the check to the `run` method, i.e.
```php
if ( this->fld_upvotes ) {
	$this->addUpvoteData( $result, $revisionIds );
}
```